src/sentry/api/endpoints/sentry_apps.py

```diff
@@ -6,16 +6,38 @@
 from sentry.api.paginator import OffsetPaginator
 from sentry.api.serializers import serialize
 from sentry.api.serializers.rest_framework import SentryAppSerializer
+from sentry.constants import SentryAppStatus
 from sentry.features.helpers import requires_feature
 from sentry.mediators.sentry_apps import Creator
 from sentry.models import SentryApp
 
 
 class SentryAppsEndpoint(SentryAppsBaseEndpoint):
     def get(self, request):
+        status = request.GET.get('status')
+
+        if status == 'published':
+            queryset = SentryApp.objects.filter(status=SentryAppStatus.PUBLISHED)
+
+        elif status == 'unpublished':
```

Comment:
When would we want unpublished integrations?

Reply:
could be useful for `_admin` ?

Reply:
also would be for people that want to see all their unpublished apps across organizations